hack/ostree_tag.sh

```diff
@@ -1,6 +1,4 @@
 #!/bin/bash
-if ! pkg-config glib-2.0 gobject-2.0 ostree-1 libselinux 2> /dev/null ; then
-	echo containers_image_ostree_stub
-else
-	echo ostree
+if pkg-config glib-2.0 gobject-2.0 ostree-1 libselinux 2> /dev/null ; then
+	echo ostree ostree_repos
```

Comment:
This will need to be updated to use `containers_image_ostree` instead of `ostree_repos`.